testingfacade/Docs/QuestionSchema.json

```diff
@@ -0,0 +1,177 @@
+                    }
+                }
+
+            ]
+        },
+        "question_id": {
+            "description": "A unique string for each individual question"
+        },
+        "name": {
+            "description": "The test name, identical to the question_id unless a test contains multiple questions"
+        },
+        "description": {
+            "description": "String, a brief description of the test"
+        },
+        "question": {
+            "description": "String, human readable question text containing instructions to carry out the test"
+        },
+        "answers": {
+            "type": "array",
+            "description": "Array containing answer objects to be used to select the correct answer_id to be returned.",
+            "examples": [
+                [
+                    {
+                        "answer_id": "answer_2",
+                        "label": "s3/wright",
+                        "description": "Mock sender 3",
+                        "id": "71ce775c-cdc5-479a-bd29-87c081905ecf",
+                        "answer_str": "s3/wright (Mock sender 3, 71ce775c-cdc5-479a-bd29-87c081905ecf)"
+                    }
+                ]
+            ],
+            "additionalItems": true,
+            "items": {
+                "anyOf": [
+                    {
+                        "description": "Individual answer details, the label, description and id of the relevant resource and an associated answer_id",
+                        "required": [
+                            "answer_id",
+                            "label",
+                            "description",
+                            "id",
+                            "answer_str"
+                        ],
+                        "properties": {
+                            "answer_id": {
+                                "description": "String, unique to each answer within a question. This is the value to be returned for validation."
+                            },
+                            "label": {
+                                "description": "Resource label as might be displayed on the NCuT"
+                            },
+                            "description": {
+                                "description": "Resource description as might be displayed on the NCuT"
+                            },
+                            "id": {
+                                "description": "Resource ID as might be displayed on the NCuT"
+                            },
+                            "answer_str": {
+                                "description": "String combining all the resource details for display on the testing facade"
+                            }
+                        },
+                        "additionalProperties": true
+                    }
+                ]
+            }
+        },
+        "time_sent": {
+            "description": "Float, time the question was sent"
```

Comment:
Like the time answered, which I think can just be tracked by the testing tool, when the Answer API is contacted, I'm not sure why this needs to go in the question request?